### PR TITLE
menual-e2e use npx and installs cocoapods dependencies

### PR DESCRIPTION
**Pull request by dulmandakh**
## Summary

This PR tries to improve manual-e2e experience by installing RNTester cocoapods dependencies automatically, and using npx to run react-native command. Removing RNTester/Pods because pod specs might change while RN version still the same or 1000.0.0, and this might cause unexpected issues.

## Changelog

[General] [Changed] - test-manual-e2e.sh script uses npx and installs cocoapods dependencies

## Test Plan

run ./scripts/test-manual-e2e.sh, and it'll install cocoapods dependencies and run npx react-native run-android.

### CODE REVIEW

File: scripts/test-manual-e2e.sh [hunk 1]
```diff
@@ -61,6 +61,8 @@ adb shell am start -n com.facebook.react.uiapp/.RNTesterActivity
 info "Press any key to open the workspace in Xcode, then build and test manually."
 info ""
 read -n 1
+success "Installing CocoaPods dependencies"
+rm -rf RNTester/Pods && cd RNTester && pod install
```
Comment (kelset) [inline scripts/test-manual-e2e.sh]: Should this command also check that the pods repos are up to date?
```suggestion
rm -rf RNTester/Pods && cd RNTester && pod install  --repo-update
```

Comment (dulmandakh) [reply scripts/test-manual-e2e.sh]: IMHO, all dependencies are local and there is no need of updating repo, therefore it usually takes some time.

**Comment by dulmandakh** [comment]
removing RNTester/Podfile.lock because it's not updated on regular basis when other specs are updated, and it's being modified almost everytime during pod install.

**Comment by dulmandakh** [comment]
@hramos done

**Comment by hramos** [comment]
These changes landed as part of https://github.com/facebook/react-native/commit/be355a24b72c7e73abb26c2d1a14831a4f7bb2d6. GitHub did not auto-close this PR because of an issue we had with the tool that keeps the repo in sync with our internal source control.

**Pull request closed without merging**